LSP-rust-analyzer.sublime-commands

```diff
@@ -26,5 +26,25 @@
     {
         "caption": "LSP-rust-analyzer: Expand Macro Recursively",
         "command": "rust_analyzer_expand_macro"
+    },
+    {
+        "caption": "LSP-rust-analyzer: View Item Tree",
+        "command": "rust_analyzer_view_item_tree"
+    },
+    {
+        "caption": "LSP-rust-analyzer: View Syntax Tree",
+        "command": "rust_analyzer_syntax_tree"
+    },
+    {
+        "caption": "LSP-rust-analyzer: Open Cargo.toml",
+        "command": "rust_analyzer_open_cargo_toml"
+    },
+    {
+        "caption": "LSP-rust-analyzer: Find Matching Brace",
+        "command": "rust_analyzer_matching_brace"
+    },
```

Comment:
It's a bit weird to have this as command. Wouldn't you want to request this implicitly? Without user action? A bit like the textDocument/documentHighlight feature in the base protocol.

Reply:
Or like in my comment about "Join Lines", maybe override default keybinding with this one if the session is active.

Reply:
removed for now